src/utils.ts

```diff
@@ -59,6 +60,26 @@ export function errorHandling(errorDetails: any, label?: string, moreInfo?: stri
     return;
 }
 
+/*************************************************************************************************************
+ * Refresh Profile and Session
+ * @param {sessNode} IZoweTreeNode
+ * @param {profile} IProfileLoaded
```

Comment:
This parameter does not exist in the type signature bellow.

Reply:
the profile? I'll update it

Reply:
Yes, the profile. The JSdoc duplicates the knowledge present i the function signature. I think it is better to delete it.